int32 JointID
int32 AttributeID
float64 Value
